Mergin/plugin.py

```diff
@@ -547,6 +571,22 @@ def unload(self):
 
         QgsApplication.processingRegistry().removeProvider(self.provider)
 
+        # unlock geodiff library and revert monkeypatching
+        if os.name == "nt":
+            try:
+                lib_path = os.path.join(os.path.dirname(__file__), "mergin", "deps", "pygeodiff")
+                for f in os.listdir(lib_path):
+                    if f.lower().endswith(".pyd"):
+                        os.remove(os.path.join(lib_path, f))
```

Comment:
now I realized - isn't this unload method also called on normal unload of QGIS (e.g. closing QGIS?) - if so this will screw the installation of the plugin? maybe it was not good idea.. very sorry ...